spec/e2e/endtoend.spec.js

```diff
@@ -34,15 +34,32 @@ describe('Cordova create and build', function () {
     var buildDirectory = path.join(__dirname, '../..');
     var appPackagesFolder = path.join(buildDirectory, projectFolder, 'AppPackages');
     var buildScriptPath = '"' + path.join(buildDirectory, projectFolder, 'cordova', 'build') + '"';
+    var silent = false;
+
+    function verifySubDirContainsFile (subDirName, fileName, count) {
+        count = typeof count !== 'undefined' ? count : 1;
```

Comment:
ugly overwrite (I think)

I would propose something like this:

```js
var expectedCount = typeof count !== 'undefined' ? count : 1;
```

Reply:
that line is actually just an ugly "hack" to get default params for older node versions, otherwise one could just `count = 1` in the param list of the functions. yay JavaScript.